src/averageAges.js

```diff
@@ -14,12 +20,16 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((man) => man.sex === 'm'
+      && Math.ceil(man.died / 100) === century)
+    : people.filter((man) => man.sex === 'm');
+
+  const result = calculateAverage(men);
+
+  return result;
```

Comment:
No need to create additional variable result, you can immediately return the result of your function

Reply:
Okay, got it. Thank you!